cmd/juju/caas/add_test.go

```diff
@@ -385,11 +385,18 @@ type regionTestCase struct {
 }
 
 func (s *addCAASSuite) TestRegionFlag(c *gc.C) {
+	errMsg := `
+	Juju needs to query the k8s cluster to ensure that the recommended
+	storage defaults are available and to detect the cluster's cloud/region.
+	This was not possible in this case so run add-k8s again, using
+	--storage=<name> to specify the storage class to use and
+	--region=<cloudType>/<region> to specify the cloud/region.
```

Comment:
--cloud-type=<cloudType> or --region=<cloudType>/<region> to specify the cloud/region.